prov/efa/src/rxr/rxr_pkt_cmd.c

```diff
@@ -562,11 +565,19 @@ void rxr_pkt_handle_recv_completion(struct rxr_ep *ep,
 		rxr_pkt_handle_rtr_recv(ep, pkt_entry);
 		return;
 	default:
-		FI_WARN(&rxr_prov, FI_LOG_CQ,
-			"invalid control pkt type %d\n",
-			rxr_get_base_hdr(pkt_entry->pkt)->type);
-		assert(0 && "invalid control pkt type");
-		rxr_cq_handle_cq_error(ep, -FI_EIO);
+		if (base_hdr->type >= RXR_EXTRA_REQ_PKT_END) {
+			FI_WARN(&rxr_prov, FI_LOG_CQ,
+				"Peer %d is requesting feature %d, which this EP does not support.\n"
```

Comment:
We could add a string coverter for our feature bitmap, maybe as a separate commit later.

Reply:
Thank you. will submit a separate PR for this